test/e2e/cluster_upgrade.go

```diff
@@ -302,19 +302,13 @@ var _ = Describe("Skipped", func() {
 						Logf("Node template %s is still in use; not cleaning up", tmplBefore)
 						return
 					}
-					// TODO(mbforbes): Distinguish between transient failures
```

Comment:
I don't think this TODO is actually necessary—we'll just retry a little on both, and that's fine.